BlueDBOnDisk/src/main/java/org/bluedb/disk/file/FileUtils.java

```diff
@@ -45,10 +48,25 @@ protected FileUtils() {}  // just to get test coverage to 100%
 		return getFolderContentsExcludingTempFiles(path.toFile(), endsWithSuffix);
 	}
 
+	public static DirectoryStream<Path> getFolderContentsExcludingTempFilesAsStream(Path path, String suffix) throws IOException {
+		FileFilter endsWithSuffix = (f) -> f.toPath().toString().endsWith(suffix);

```

Comment:
Shouldn't this filter use the IS_NOT_TEMP_FILE filter in addition to the ends with filter in order to exclude the temp files?

Reply:
Oh, nvm. It calls a method that adds the is not temp... sorry.